src/main/target/REVOLT/target.h

```diff
@@ -0,0 +1,161 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "RVLT"
```

Comment:
There is the expectation that there is a 1 : 1 mapping between board identifiers and target names, so we should be doing something like

```
#if defined(REVOLTOSD)
￼#define TARGET_BOARD_IDENTIFIER "RVLO"
#else
￼#define TARGET_BOARD_IDENTIFIER "RVLT"
#endif
```